packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -286,6 +297,80 @@ skipInNode(describe)('plugin-meetings', () => {
           throw e;
         }));
 
+      // FHD - remote video tests - starts
+      // Firefox bug - https://bugzilla.mozilla.org/show_bug.cgi?id=1537986
+      if (getBrowserName() !== 'Firefox') {
+        const getRemoteHeight = (minHeight = 360, maxHeight = 1080) => new Promise((resolve, reject) => {
+          testInterval = setInterval(() => {
+            const remoteSettings = bob.meeting.mediaProperties.remoteVideoTrack.getSettings();
+            const height = remoteSettings.height && +remoteSettings.height;
+
+            console.log(`Resolution for ${minHeight} ${maxHeight} : `, count, height);
+            if (height >= minHeight && height <= maxHeight) {
+              clearInterval(testInterval);
+              resolve(height);
+            }
+
+            if (count === INTERVAL_COUNT) {
+              clearInterval(testInterval);
+              if (!height) reject(new Error('Resolution undefined'));
+              else resolve(height);
+            }
+
+            count += 1;
+          }, WAIT_TIME);
+        });
+
+        it('alice resolution is 1080p, check what bob receives', async () => {
+          await testUtils.delayedPromise(alice.meeting.setLocalVideoQuality('1080p'));
+          const height = await getRemoteHeight(1080);
+
+          assert.isAbove(height, 500);
+          assert.isBelow(height, 1081);
+        });
+
+        it('bob requests 480p, should get resolution around 480p', async () => {
```

Comment:
Has this test ever passed for you? I see it's failing in CI and I don't think it will ever pass, because that functionality is broken, I think it can only pass by luck if the network conditions restrict the resolution - see a fix for this done in [this PR](https://sqbu-github.cisco.com/WebExSquared/webrtc-media-core/pull/139) in the webrtc-media lib. 
BTW, I don't think we should be fixing this issue now in SDK, we should just wait for SDK v3 which uses the webrtc-media lib that has the fix and until then this test should be disabled.

Reply:
Skipped

Reply:
if it's skipped, it should have a comment pointing to a jira describing the issue, so that once that jira is resolved, the test can be enabled.
Having a skipped test with no comment and no jira means that it will be skipped forever, so it will be useless and we may as well just delete it